src/app/shared/components/carousel/carousel2.0.html

```diff
@@ -1,46 +1,51 @@
-<!-- Container for the image gallery -->
-<div class="container">
-
-    <!-- Full-width images with number text -->
-    <div class="contentSlides">
-        <div class="mySlides">
-          <div class="numbertext">1/4</div>
-            <img  src="https://d1ih8jugeo2m5m.cloudfront.net/2021/06/Ecommerce-Thumbnail.jpg" style="width:100%" alt="">
+<div id="carouselExampleCaptions" class="carousel slide" data-bs-ride="false">
+  <div class="carousel-indicators" >
+    <!-- *ngFor="let data of datacarousel; let i=index" -->
+    <button type="button" (click)="showSlide(0)" data-bs-target="#carouselExampleCaptions" data-bs-slide-to="i" class="carousel-indicator" aria-current="true" aria-label="Slide 1"></button>
+    <button type="button" (click)="showSlide(1)"  data-bs-target="#carouselExampleCaptions" data-bs-slide-to="1" class="carousel-indicator" aria-label="Slide 2"></button>
+    <button type="button" (click)="showSlide(2)" data-bs-target="#carouselExampleCaptions" data-bs-slide-to="2" class="carousel-indicator" aria-label="Slide 3"></button>
+    <button type="button" (click)="showSlide(3)" data-bs-target="#carouselExampleCaptions" data-bs-slide-to="3" class="carousel-indicator" aria-label="Slide 4"></button>
+ 
+  </div>
+  <div class="carousel-inner">
+    <div class="wapper-carousel" *ngFor="let data of datacarousel">
```

Comment:
when using the carousel in this way, this is how it is displayed on the page:
![Screenshot from 2022-12-08 14-30-17](https://user-images.githubusercontent.com/74382484/206539374-5a8677cc-0b1d-4118-82a9-d25d62329e95.png)
![Screenshot from 2022-12-08 14-30-03](https://user-images.githubusercontent.com/74382484/206539401-317198d1-c037-4990-b8a7-1bdd87e95ed4.png)